Habinezajanvier/Anes_Pro
# Setting eslint #172283759

#### What does this PR do
* Setting eslint for linting
#### Description of Task to be completed?
* set Eslint 
* set prettier
#### How should this be manually tested?
* clone the repo `git clone ....`
* checkout the branch `git checkout ch-setting-eslint-#172283759`
* try to edit any js file
* you should see some violation warnings by eslint
#### Any background context you want to provide?
* Eslint and prettier extension should be installed on text editor, otherwise it will not work
#### What are the relevant pivotal tracker stories?
#172283759
